### PR TITLE
Adding academy links and meetings links for people who get stuck 

### PR DESCRIPTION
**Pull request by Hafflgav**
## What is the purpose of the change
* Create a “getting stuck? Book a meeting with one of our reps” link on the getting started guide, which directs them to our schedule to book a meeting. (Maybe a crazy wild idea, as it could lead to an overload of meetings, but who knows? Maybe that doesn’t happen and we will have quality conversations which might also end in SQO’s)
* Link to the Camunda Academy whenever possible

Links to https://github.com/camunda/product-hub/issues/375

## Are there related marketing activities
* no 

## When should this change go live?
* As soon as possible - but no rush 

## PR Checklist

- [X] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [X] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [no] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by Hafflgav** [comment]
Hey @christinaausley, 
even there are not a lot of changes I would appreciate you taking a look at them. :) 
Thanks a lot in advance!

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 1]
```diff
@@ -12,11 +12,14 @@ When a process instance arrives at such a user task, a new job is created. The p
 
 While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial.
 
-In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser.
+In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser. If you rather prefer a video based learning experience we got you covered as well. [This Camunda Academy course](https://bit.ly/3PJJocB) features the same content which is being discussed in this guide.
```
Comment (christinaausley) [inline docs/guides/getting-started-orchestrate-human-tasks.md:15]: Same comment here on accessibility. I'd rather not include it if not everyone is able to access it.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 2]
```diff
@@ -12,11 +12,14 @@ When a process instance arrives at such a user task, a new job is created. The p
...
 
 :::note
-New to BPMN or want to learn more? Visit our [BPMN cheat sheet](https://page.camunda.com/wp-bpmn-2-0-business-process-model-and-notation-en) for an overview of all BPMN symbols.
-:::
+Getting stuck? Book a meeting with our User Success Specialists. Follow the links dependent on the location you are joining:
+
+- [American regions](https://meetings.salesloft.com/camunda/jonathanjohnson)
```
Comment (christinaausley) [inline docs/guides/getting-started-orchestrate-human-tasks.md:20]: Same comment here as above.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 3]
```diff
@@ -90,6 +94,7 @@ In this case, the token will move through the gateway and (according to the cond
 
 ## Additional resources and next steps
 
+- [Video-based Human Task Orchestration Course](https://bit.ly/3PJJocB)
```
Comment (christinaausley) [inline docs/guides/getting-started-orchestrate-human-tasks.md:97]: Same comment here 👍

**Review by christinaausley — CHANGES_REQUESTED** [review]
Just a few comments for clarification before moving forward, largely surrounding the following:

Do all individuals reading this documentation have access to the mentioned Camunda Academy sources? I just want to ensure these links are accessible to everyone before merging so there isn't confusion if certain groups cannot access these links.

Feel free to take a look at my latest commit for changes I've made, including:
- Camunda Platform 8 rather than Camunda 8
- Camunda Academy rather than Academy

**Comment by akeller** [comment]
This is right on the edge of my comfort level for this kind of CTA in the docs. This could discourage people to try the product on their own (meaning, it's perceived as being too difficult without guided assistance beyond the docs). If this experiment is successful, I would advocate moving this CTA to the bottom of the page rather than the top.

File: docs/guides/getting-started-orchestrate-api-endpoints.md [hunk 4]
```diff
@@ -13,6 +13,13 @@ The concept of a connector consists of two parts: the business logic is implemen
 
 This guide will walk you through working with a REST connector task as a first time Camunda Platform 8 user.
 
+:::note
+Getting stuck? Book a meeting with our User Success Specialists. Follow the links dependent on the location you are joining:
+
+- [American regions](https://meetings.salesloft.com/camunda/jonathanjohnson)
```
[the remark this answers is not in the document]
Comment (Hafflgav) [reply docs/guides/getting-started-orchestrate-api-endpoints.md:19]: Yes this link is for the whole American continent

File: docs/guides/getting-started-orchestrate-api-endpoints.md [hunk 5]
```diff
@@ -56,3 +63,4 @@ To deploy your process, take the following steps:
 ## Additional resources and next steps
 
 - Dive deeper into [connectors](/components/integration-framework/introduction.md#connectors).
+- Learn more about Camunda Platform 8 and what it can do by reading [What is Camunda Platform 8](/components/concepts/what-is-camunda-platform-8.md) or watching a [platform overview video](https://bit.ly/3TjNEm7) in our Academy.
```
[the remark this answers is not in the document]
Comment (Hafflgav) [reply docs/guides/getting-started-orchestrate-api-endpoints.md]: Yes, the Academy video is accessible to everyone. 👍

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 1 above)
Comment (Hafflgav) [reply docs/guides/getting-started-orchestrate-human-tasks.md]: Yes, the Academy video is accessible to everyone. 👍

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 3 above)
Comment (Hafflgav) [reply docs/guides/getting-started-orchestrate-human-tasks.md:97]: Yes, the Academy video is accessible to everyone. 👍

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 2 above)
Comment (Hafflgav) [reply docs/guides/getting-started-orchestrate-human-tasks.md:20]: Yes, this link is valid for the whole American continent. :)

**Comment by Hafflgav** [comment]
Thanks @christinaausley for your review and changes! 💯 

I can understand why this is right on the edge of your comfort level. Due to this I would keep the experiment period as short as possible. Can you accept a runtime of max 2 weeks after this is being released? 
It is also valid if you do not want to have this CTA featured in the docs at all - just let me know. :)

**Comment by akeller** [comment]
I'm good with 2 weeks @Hafflgav. Please let me know the results of the experiment when it goes live.

**Comment by Hafflgav** [comment]
Will do @akeller! 💯 
Is there anything I need to change. Currently the merging is still blocked @christinaausley. 🤔

**Review by christinaausley — APPROVED** [review]

**Pull request merged**